app/controllers/import_controller.rb

```diff
@@ -274,6 +276,26 @@ def validate_assigned_user(value, row_ind)
     raise ValidationError.new("'#{value}' is not valid for 'Assigned User', acceptable values are numbers between 1-9999", row_ind)
   end
 
+  def validate_workflow_specific_enums(workflow, field, value, row_ind)
+    return nil if value.blank?
+
+    normalized_value = unformat_enum_field(value)
+    if workflow == :exposure
+      return NORMALIZED_EXPOSURE_ENUMS[field][normalized_value] if NORMALIZED_EXPOSURE_ENUMS[field].keys.include?(normalized_value)
+
+      # rubocop:disable Layout/LineLength
```

Comment:
Question for reviewers - The line length for line 287 was triggering Rubocop. Is it temporary disable appropriate to appease rubocop's line length check in this instance?

Reply:
We typically avoid doing that in the application code (we do it in some places in the tests), so I'd ask you to try and avoid doing so here too if possible (which I think should be do-able).

Reply:
Just pushed an update splitting the line up.